exporter/metric/metric.go

```diff
@@ -643,8 +677,49 @@ func histToDistribution[N int64 | float64](hist metricdata.HistogramDataPoint[N]
 				},
 			},
 		},
-		// TODO: support exemplars
+		Exemplars: toDistributionExemplar[N](hist.Exemplars),
+	}
+}
+
+func expHistToDistribution[N int64 | float64](hist metricdata.ExponentialHistogramDataPoint[N]) *distribution.Distribution {
+	numFiniteBuckets := len(hist.PositiveBucket.Counts)
+	counts := make([]int64, numFiniteBuckets)
+	positiveBucketCounts := 0
+	growthFactor := math.Exp2(math.Exp2(-float64(hist.Scale)))
+	for i, v := range hist.PositiveBucket.Counts {
+		counts[i] = int64(v)
```

Comment:
[Opentelemetry Exponential Buckets](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exponential-buckets) support `Negative Indices` too, but AFAIU the [TypedValue Exponential Buckets](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue#exponential) only support `Positive Indices`.

Reply:
Thats correct.  All negative observations and zero observations should be in the underflow bucket.  See https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/87124ac8635698cc7cdc9f8f2dfca94a8aa13606/exporter/collector/metrics.go#L989-L993

Reply:
Fixed. I used as reference the implementation of collector/metrics.go : exponentialHistogramPoint to reimplement all the mentioned details of expHistToDistribution in this review, but using the correct types and methods.